flask/boxwise_flask/models/box.py

```diff
@@ -110,38 +110,20 @@ def __unicode__(self):
         return self.box_label_identifier
 
     @staticmethod
-    def create_box(box_creation_input):
+    def get_box(box_id):
+        return Box.get(Box.box_label_identifier == box_id)
```

Comment:
looking at the old deleted stuff, box_label_idenfier is not the primary key. I believe @spaudanjo in his implementation put it as box_short_uuid which is a clever solution. I'll leave it to the two of you to determine which makes more sense.

Reply:
I will also say @spaudanjo comments on the business logic are quite good, you might want to keep them @pylipp

Reply:
I'm aware that `box_label_identifier` is not the PK. However it is used here because of the way the query `box(boxId: String)` is defined. If it makes more sense to have `box(id: ID)` I'm happy to change the query (but I think on the FE the box label identifier is requested as an input to look a box), or to simply rename the field to e.g. `box_uuid`

The comments about the business logic were part of the code since the days it was still in a separate `boxwise-flask` repo :D I don't think there have to be comments in the BE/FE referencing any possibilities or options what to do on the other end, that should be part of the backlog rather than staying in the code base (and potentially becoming stale). However I did check the comments again, and noticed an important aspect about the length of the box_label_id field, thank you!

Reply:
@aerinsol It looks like box_id is indexed with a unique constraint, so it should be fine but I concur with @pylipp comments about fixed length and @spaudanjo nice suggestion for using uuid, but it will likely affect DropApp. As a result, if we decide to change to uuid, we will have to verify DropApp will work well.

Reply:
@pylipp @vahidbazzaz `box_label_identifier` is required as an input to retrieve boxes, that's true. However, it's likely we'll have to request the internal PK for the `box` type as well. Therefore I'd prefer it if there was some indication that the id being used is for the label vs. the internal PK.

Reply:
> I'm aware that `box_label_identifier` is not the PK. However it is used here because of the way the query `box(boxId: String)` is defined. If it makes more sense to have `box(id: ID)` I'm happy to change the query (but I think on the FE the box label identifier is requested as an input to look a box), or to simply rename the field to e.g. `box_uuid`
> 

Sorry, I think you are right, it's fine as is. Realized this is in the models section of the codebase where you are connecting the model definitions with the true MySQL back-end, I misread it and thought it was having to do with the GraphQL definitions.

Please keep as is and carry on! @pylipp